android-patches/patches/OfficeRNHost/ReactAndroid/src/main/java/com/facebook/react/bridge/ReactBridge.java

```diff
@@ -0,0 +1,30 @@
+--- /dev/code/rnm-66-fresh/ReactAndroid/src/main/java/com/facebook/react/bridge/ReactBridge.java	2022-02-13 19:54:48.563686391 -0800
++++ /dev/code/react-native-macos/ReactAndroid/src/main/java/com/facebook/react/bridge/ReactBridge.java	2022-02-13 22:53:50.732054489 -0800
+@@ -31,6 +31,27 @@
+     Systrace.beginSection(
+         TRACE_TAG_REACT_JAVA_BRIDGE, "ReactBridge.staticInit::load:reactnativejni");
+     ReactMarker.logMarker(ReactMarkerConstants.LOAD_REACT_NATIVE_SO_FILE_START);
++
++    // JS Engine is configurable.. And we exepct only one packaged
++    // Hence ignore failure
++    try {
++	SoLoader.loadLibrary("hermes");
++    }catch (UnsatisfiedLinkError jscE){}
++
++    try {
++	SoLoader.loadLibrary("v8jsi");
++    }catch (UnsatisfiedLinkError jscE){}
++
++    SoLoader.loadLibrary("glog");
++    SoLoader.loadLibrary("glog_init");
++    SoLoader.loadLibrary("fb");
++    SoLoader.loadLibrary("fbjni");
++    SoLoader.loadLibrary("yoga");
++    SoLoader.loadLibrary("folly_json");
++    SoLoader.loadLibrary("reactperfloggerjni");
++    SoLoader.loadLibrary("jsinspector");
```

Comment:
does this need to be loaded in all scenarios? What about release builds?

Reply:
Yeah.. these dependencies comes from OSS builds .. and they are not flavor specific .. This are lots of room for improvement in the native builds in OSS ..